Sources/TokamakDOM/Views/Buttons/Button.swift

```diff
@@ -17,10 +17,30 @@
 
 import TokamakCore
 
-extension Button: ViewDeferredToRenderer where Label == Text {
+extension _Button: ViewDeferredToRenderer where Label == Text {
   public var deferredBody: AnyView {
-    AnyView(HTML("button", listeners: ["click": { _ in _ButtonProxy(self).action() }]) {
-      _ButtonProxy(self).label.subject
+    let attributes: [String: String]
+    if buttonStyle.type == DefaultButtonStyle.self {
+      attributes = [:]
+    } else {
+      attributes = ["style": """
+      -webkit-appearance: none;
+      -moz-appearance: none;
+      appearance: none;
```

Comment:
A few more things to reset. It looks like the default text size on native inside buttons matches the size of labels. However, the `<button>` on web is physically smaller and there’s no way to scale it up :(

```suggestion
      -webkit-appearance: none;
      -moz-appearance: none;
      appearance: none;
      background: transparent;
      border: none;
      margin: 0;
      padding: 0;
      font-size: inherit;
```